src/store/modules/routingSlip.ts

```diff
@@ -253,4 +271,41 @@ export default class RoutingSlipModule extends VuexModule {
       return response.data
     }
   }
+
+  @Action({ rawError: true })
+  public async getLinkedRoutingSlips (routingSlipNumber): Promise<void> {
+    try {
+      const response = await RoutingSlipService.getLinkedRoutingSlips(routingSlipNumber, true)
+      const context: any = this.context
+      if (response && response.data && response.status === 200) {
+        /* The response to getLinkedRoutingSlips consists of following values:
+        1. children: non empy, parent: empty: which implies that the routingslip is a parent slip
+        2. children: empty, parent: non empty: which implies that the slip is a child slip
+        3. no content 204 resposne: non linked routingslip where we need to reset the states
+        */
+        if (response.data?.children) {
+          const linkedRoutingSlips: LinkedRoutingSlips = {
+            children: response.data?.children,
```

Comment:
can we remove if else and refactor?

Reply:
yeah . Thanks, we have totally removed if :-P